index.html

```diff
@@ -1,17 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Movie Quiz</title>
+    <link rel="stylesheet" href="styles/style.css" />
```

Comment:
Nie ma potrzeby ładowania styli tutaj jeśli korzysta się z Vite, on to doda sam z siebie, na podstawie importu w pliku JS

Reply:
Już poprawiam

Reply:
Ustawiłem na import w innych html-ach ale nie ładują się style, czy wszystkie importy do main,js czy w innych html już bezposrednio przez link?

Reply:
Okej już poprawione :)

Reply:
Build powstał ale nie po kliku w actors nie kieruje - sprawdzę w docsach jeszcze ale wydaje mi się że vite akceptowało multipage apps  z innymi plikami w folderze, chyba że netlify blokuje - tu nie wiem, bo korzystałem z gh-pages dotychczas